src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,6 +6,7 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_UID = new Prefix("id/");
```

Comment:
Should we use a single character to denote id to keep it in line with the other prefixes?

Reply:
I don't have a preference, however, I think that word id is succinct enough.